Sources/PostgresNIO/New/PostgresCodable.swift

```diff
@@ -123,6 +163,46 @@ extension PostgresNonThrowingEncodable {
     }
 }
 
+extension PostgresDynamicTypeThrowingEncodable {
```

Comment:
We should be able to remove the implementations L126-L164 here. Those new impl. replace the old ones.